lib/Workflow/Factory.pm

```diff
@@ -262,9 +262,9 @@ sub _add_workflow_config {
         $log->is_info
             && $log->info("Added all workflow states...");
 
-        $self->_load_observers($workflow_config);
+        my $num_observers = $self->_load_observers($workflow_config);
         $log->is_info
-            && $log->info("Added all workflow observers...");
+            && $log->info("Added $num_observers workflow observers...");
```

Comment:
Doesn't `_load_observers()` log this already?

Reply:
Hmm one could argue the observer logging belongs to the level below the logging happening at the level executed at `add_config`.

I have removed the observer logging information from: `_add_workflow_config` and refined it in `_load_observers` based on your feedback.